packages/astro/test/static-build.test.js

```diff
@@ -90,7 +94,7 @@ describe('Static build', () => {
 			const links = $('link[rel=stylesheet]');
 			for (const link of links) {
 				const href = $(link).attr('href');
-				const data = await fixture.readFile(addLeadingSlash(href));
+				const data = await fixture.readFile(removeBasePath(addLeadingSlash(href)));
```

Comment:
Hmm... how does this work? Are the assets emitted at `/subpath/assets` or `/assets`?

Reply:
Assets are built to `/dist`, the main `astro build` bug here was that injected scripts and links didn't include the subpath

This test finds a stylesheet similar to `<link rel="stylesheet" href="/subpath/assets/chunk.css">` and strips off the subpath when verifying the file exists (mimicing the behavior once it's deployed to a subpath in production)